product_supplierinfo_for_customer_sale/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+    :target: http://www.gnu.org/licenses/agpl-3.0.en.html
+
+==========================================
+Product supplier info for customers - Sale
+==========================================
+
+This module extends of module *Product supplier info for customers* and allows
+use the product codes and product name configured in each products in sale
+orders.
+
+Installation
+============
+
+To install this module, you need:
```

Comment:
Change this description with: "Installing the module sale and product_supplier_info_for_customer this module will be installed"